eman2/convert/dataimport.py

```diff
@@ -31,7 +31,9 @@
 import pyworkflow.utils as pwutils
 from pwem.objects.data import Coordinate, CTFModel
 from pwem.objects.data_tiltpairs import Angles
-from pwem.emlib.metadata import MetaData, MDL_XCOOR, MDL_YCOOR, MDL_PICKING_PARTICLE_SIZE
+from pwem.emlib.metadata import (MetaData, MDL_XCOOR, MDL_YCOOR, MDL_ZCOOR,
+                                 MDL_PICKING_PARTICLE_SIZE)
+from eman2.constants import TOMO_NEEDED_MSG
```

Comment:
this can go

Reply:
Fixed in https://github.com/scipion-em/scipion-em-eman2/pull/64/commits/3b805869019181e3e364f9837a73ef99f09c12dd